sports_association_lorena/models/sport_issue.py

```diff
@@ -16,6 +16,12 @@ class SportIssue(models.Model):
     tag_ids = fields.Many2many('sport.issue.tag', string='Tags')
     color = fields.Integer(string='Color', default=0)
     cost = fields.Float(string='Cost')
+    assigned = fields.Boolean(string='Assigned', computed='')
+
+    def _computed_assigned(self):
+        for record in self:
+            if user_id:
+                self.assigned = True
```

Comment:
Recuerda, si usas el bucle for record in self, luego en vez de self.assigned ponle record.assigned